.config/sample-config.json

```
@@ -43,6 +43,38 @@
             ],
             "thumbnailPath": "assets/thumbnail.png",
             "suggested": false
+        },
+        {
+            "id": "excel-add-in-chartapi-analyze-data",
+            "onboardDate": "2024-11-11",
+            "title": "Use the Excel Chart APIs to analyze data",
+            "description": "The sample add-in demonstrates Excel add-in capabilities to help users using chart API to analyze data.",
```

Let's make this less about the technical details and more focus on the scenario.
```suggestion
            "description": "The sample Excel add-in shows how to use charts to help analyze data.",
```